README.md

```diff
@@ -74,5 +75,49 @@ See [this demo](https://chalu.github.io/int-range/index.html) from [the docs fol
       sequence: multiples({ of: 5, stepsOf: 2 })
     });
     ```
+
+## Extensibility
+
+Have need for a custom or domain specific integer sequence? You simply need to make a validator function!
+
+1. Import the util's sequencer engine
+    ```javascript
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/manual/chalu/int-range/pullRequest?prid=5136855)